content/en/join-our-team/_index.html

```diff
@@ -5,103 +5,106 @@
+
+                <p>CDS welcomes all applicants, including Veterans and people of all races, ethnicities, religions, sexual orientations, gender identities and expressions, national origins, disabilities, ages, body sizes and including those with diverse households and family commitments.</p>
             </div>
         </div>
     </div>
 </section>
 
-<section class="work-with-us--section">
+<section class="work-with-us--image">
      <img data-src="https://de2an9clyit2x.cloudfront.net/team_photo_outdoor_8e5bf4735c.jpg" data-srcset="https://de2an9clyit2x.cloudfront.net/team_photo_outdoor_8e5bf4735c.jpg" src="/img/cds/placeholder.gif" class="lazy img-fluid" alt="CDS Team">
 </section>
 
-<section class="work-with-us--section">
+<section class="work-with-us--section bg-white">
     <div class="container work-with-us--text">
-        <div class="page-wide--outer-container-padding">
-            <div class="row wb-eqht">
-                <div class="col-sm-10 col-sm-offset-1 border-top">
-                    <h3>What we’re offering</h3>
-                </div>
-                <div class="col-sm-10 col-sm-offset-1 mrgn-bttm-lg">
-                    <ul>
-                        <li><span>An agile, dynamic team where you can work alongside experts from many digital
-                                disciplines.</span></li>
-                        <li><span>An enabling environment with access to the modern tools you need to do great work.</span></li>
-                        <li><span>A laser-focus on people's needs with the ability to see the impact of your efforts.</span></li>
-                        <li><span>Flexible work arrangements that ensure strong work-life balance.</span></li>
-                    </ul>
+        <div class="row">
+            <div class="col-sm-8 col-xs-12 border-top">
+                <h3>Our values</h3>
+            </div>
+            <div class="col-sm-8 col-xs-12 mrgn-bttm-lg">
+                <ul class="work-with-us--list">
+                    <li>Put people at the heart of services.</li>
+                    <li>Deliver measurable outcomes.</li>
+                    <li>Do the hard work to make things easier.</li>
+                    <li>Build for learning and iteration.</li>
+                    <li>Work in the open to help clear a path.</li>
+                    <li>Take care of each other.</li>
+                </ul>
+                
+                <div class="button-arrow">
+                    <a href="/our-values/">More about our values<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path d="M8 9H0V7H8V0L16 8L8 16V9Z"/></svg></a>
```

Comment:
Usually for svg's you'd add a role="img" so that screen readers don't read out all the inner elements, but since this is for presentation you should probably add role="presentation" but axe isn't complaining so /shrug

Reply:
Thanks for spotting this and the headings, and taking a look over the draft PR ✨ I'll make sure to correct it before it goes up for final reviews!